.circleci/config.yml

```diff
@@ -14,29 +14,17 @@ jobs:
   test_macos:
     macos:
       xcode: "15.3.0"
+    resource_class: macos.m1.medium.gen1
     steps:
       - checkout
       - run:
-          name: install openssl, pyenv
+          name: build hokusai
           command: |
-            brew install openssl pyenv
-            echo -e '\neval "$(pyenv init --path)"' >> ~/.bash_profile
-      - run:
-          name: install python
-          command: |
-            pyenv install 3.10
-            pyenv local 3.10
-            pip install --upgrade pip
-      - run:
-          name: install awscli
-          command: |
-            curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
-            sudo installer -pkg AWSCLIV2.pkg -target /
-      - run: make hokusai
-      - run: pyenv rehash
```

Comment:
the above steps are repeated 3x for each MacOS job (test, release-beta, release), so let's move them to a script.

Reply:
If they're only repeated _in this file_, you might be able to just use a YAML alias instead of a new file.